src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -124,6 +126,19 @@ const TreeItem: React.FC<TreeItemProps> = props => {
     return style;
   }, [isMatchItem]);
 
+  React.useEffect(() => {
+    let mounted = true;
```

Comment:
Do we really need to check for mounted?

Reply:
Hey mate yes I saw the memory leak when program starts we need to check this